docs/plugins/metrics.md

```diff
@@ -255,61 +266,81 @@ mediaMetrics.canplay({ currentTime: ... })
 ```
 
 #### Ended
+Playback has stopped because the end of the media was reached.
 
 ```js
 mediaMetrics.canplay({ currentTime: ... })
 ```
 
 #### Pause
+Playback has been paused.
 
 ```js
 mediaMetrics.pause({ currentTime: ... })
 ```
 
 #### Play
+Playback has started either initially, or after being paused.
 
 ```js
 mediaMetrics.play({ currentTime: ... })
 ```
 
+#### Progress
+Called at regular intervals to update the current progress of playback.
+
+```js
+mediaMetrics.progress({ currentTime: ... })
+```
 
 #### Suspend
 
 ```js
 mediaMetrics.suspend({ currentTime: ... })
 ```
 
-
 #### Volume Change
+Either the volume or the mute state has changed.
 
 ```js
 mediaMetrics.volumechange({ currentTime: ... })
 ```
 
 #### Waiting
+Playback has stopped because the next frame is not available, but will continue once it is available.
 
 ```js
 mediaMetrics.waiting({ currentTime: ... })
 ```
 
 #### Seeking
+Seeking was initiated.
 
 ```js
 mediaMetrics.seeking({ currentTime: ... })
 ```
 
 #### Seeked
+Seeking has completed.
 
 ```js
 mediaMetrics.seeked({ currentTime: ... })
 ```
 
 #### Ratechange
+The playback rate has changed.
 
 ```js
 mediaMetrics.ratechange({ currentTime: ... })
 ```
 
+#### Rendition
+The playback rendition, e.g. bitrate, dimensions, etc., has changed.
+
+```js
+mediaMetrics.rendition({ currentTime: ..., bitrate: ..., width: ..., height: ... })
+```
+
```

Comment:
Not 100% sure about the naming of this metric.

For reference, HLS.js calls this event 'level switching'. Shaka refers to it as an 'adaptation event'.
Would something like `qualitychange` fit here?

Any opinion on this @wouterlucas or @zombi6?

Reply:
rendition is pretty common language for both DASH & HLS:
HLS: https://tools.ietf.org/html/rfc8216
DASH: https://dashif.org/docs/workshop-2019/03-Reznik-DASH-IF-workshop-2019-CAE.pdf

Quality isn't right, because there are lots of attributes you can change across renditions, not just quality.

Adaptation works as well, but I see the word "rendition" in video context far more often.

Reply:
Cool! I'm not completely familiar on the terminology for this, so just wanted to double check on using the correct naming. But I guess rendition makes sense.

Might want to change it to `renditionchange` (similar to the html5 standard `ratechange`)?

Reply:
I'm cool with either `rendition` or `renditionChanged` although it might be nice if we could camel-case all of the multi-word methods in the API.

Reply:
Great! `renditionchanged` it is then :)

On the subject of camelCasing: the current media metrics API is designed to be consistent with the html5 media events (which are all lowercase). This allows for direct (and more performant!) integration with the events emitted by the video tag.

I do agree that camelCase is more readable in general, so I've made a note to do a test next week to see whether we could support camelCase as well, without too much impact.

Reply:
Cool, I'll make the "change" 😆.
We both mistyped in our last two comments... it's "change" (as you said originally) not "changed" assuming that's what we both meant for consistency.

Reply:
oops, fixed.